census/src/main/java/io/grpc/census/CensusTracingModule.java

```diff
@@ -231,6 +226,21 @@ private void recordMessageEvent(
     span.addMessageEvent(eventBuilder.build());
   }
 
+  private void recordAnnotation(
+      Span span, MessageEvent.Type type, int seqNo, boolean isCompressed, long size) {
+    StringBuilder messageTypeBuilder = new StringBuilder("compressed");
+    if (!isCompressed) {
+      messageTypeBuilder.insert(0, "un");
+    }
+    Map<String, AttributeValue> attributes = new HashMap<>();
+    attributes.put("id", AttributeValue.longAttributeValue(seqNo));
+    attributes.put("type", AttributeValue.stringAttributeValue(messageTypeBuilder.toString()));
+
+    String messageDirection = type.name().equalsIgnoreCase("Sent") ? "↗ " : "↘ ";
```

Comment:
Why compare based on the name instead of enum value? Especially with equalsIgnoreCase? `type == MessageEvent.Type.SENT`

Reply:
I thought using name make its more readable, but since enum itself is clear updated to compare enum values instead.